backend/src/executors.py

```diff
@@ -15,25 +15,35 @@
 class ProtBertExecutor(Executor):
     """ProtBERT executor: https://huggingface.co/Rostlab/prot_bert"""
 
+    __model = None
+
+    __tokenizer = None
+
```

Comment:
What is the convention for double-underscore variables?

Reply:
Coming from a OOP terminology, double-underscore represents private variables (do not access), while single-underscores represent protected variables.